src/System.IO.Compression/tests/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Unix;
       netstandard-Windows_NT;
+      netcoreapp-Unix;
```

Comment:
@ianhays is correct. We have https://github.com/dotnet/corefx/issues/15667 to try and run explicitly the netstandard test configurations as well but in general those should be included in the netcoreapp configurations.